lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1989,12 +1991,23 @@ SwiftASTContext::CreateInstance(lldb::LanguageType language, Module &module,
   {
     LLDB_SCOPED_TIMERF("%s (getStdlibModule)", m_description.c_str());
     const bool can_create = true;
+    std::unique_ptr<lldb_private::Progress> progress = std::make_unique<lldb_private::Progress>("Importing Swift standard library modules");
```

Comment:
`module`. AFAIK there is only one `Swift` module that is being imported there.

Reply:
But IIUC, the swift standard library module can have dependencies that are also loaded implicitly right ?

Reply:
This works when the progress reports have details (e.g. `Importing Swift module: Foundation`) but I think it would make the overall title a little less clear (i.e. having one main progress report that says `Importing Swift module` even though multiple individual imports are about to be performed). I can change it but maybe we also have an overall title and then an `Increment` title in addition to having `details`?

Reply:
I agree with Chelsea here: From a user perspective, it will look like we're only importing a single module, where actually that one module import trigger many other modules to be imported. I think `modules` makes more sense here.

Reply:
Swift has 1 dependency: SwiftShims, which isa Clang module. So I guess, you're right.